Gemfile

```diff
@@ -1,25 +1,26 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.5.3'
+ruby '3.1.2'
 
-gem 'autoprefixer-rails'
-gem 'bootsnap', '>= 1.4.2', require: false
-gem 'bootstrap-sass', '~> 3.4.1'
-gem 'devise'
-gem 'faker', :git => 'https://github.com/stympy/faker.git', :branch => 'master'
+gem 'autoprefixer-rails' 
+gem 'bootsnap', '>= 1.4.2', require: false 
+gem 'bootstrap-sass', '~> 3.4.1' 
+gem 'devise' 
+gem 'faker', :git => 'https://github.com/stympy/faker.git', :branch => 'main'
 gem 'font-awesome-sass', '~> 5.6.1'
 gem 'jquery-rails'
 gem 'jbuilder', '~> 2.7'
 gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.1'
-gem 'rails', '~> 6.0.3', '>= 6.0.3.2'
+gem 'rails', '~> 7.0.4.1' 
```

Comment:
Can we relocate the rails gem to line 5 just below the ruby version (just cause we're adjusting the file)

Reply:
Done.